xds/xds.go

```diff
@@ -16,8 +16,12 @@
  *
  */
 
-// Package xds contains xds implementation. Users need to import this package to
-// get all xds functionality.
+// Package xds contains an implementation of the xDS suite of protocols, to be
+// used by gRPC client and server applications.
+//
+// On the client-side, users simply need to import this package to get all xDS
+// functionality. On the server-side, users need to use the GRPCServer type
+// exported by this package instead of the regular grpc.Server.
 //
 // See https://github.com/grpc/grpc-go/tree/master/examples/features/xds for
```

Comment:
Will we also need a server example?  (Maybe later?)

Reply:
Sometimes github beats me totally. I don't understand why I can't reply to the above comment.

> Sure, let's go with that (hardcode to "tcp" for now).
Will do.

> Will we also need a server example? (Maybe later?)
Currently since the server doesn't have much functionality, I think we can wait till the other security integration work is complete and then we will have an example.